apps/server/src/modules/task/controller/task.controller.ts

```diff
@@ -95,4 +95,37 @@ export class TaskController {
 		const dto = CopyMapper.mapToResponse(copyStatus);
 		return dto;
 	}
+
+	@Get(':taskId')
+	async find(@Param() urlParams: TaskUrlParams, @CurrentUser() currentUser: ICurrentUser): Promise<TaskResponse> {
+		const taskWithSatusVo = await this.taskUc.find(currentUser.userId, urlParams.taskId);
+
+		const response = TaskMapper.mapToResponse(taskWithSatusVo);
+		return response;
+	}
+
+	@Post()
+	async create(@Body() params: TaskCreateParams, @CurrentUser() currentUser: ICurrentUser): Promise<TaskResponse> {
```

Comment:
This name looks looks like a generic `create` method. When looking at the ticket I think this is meant as a special kind of create operation - "create an individual task". I think we should name it like that. Another idea would be to create an `IndividualTaskUC` and put all related methods there.